include/proc_helper.h

```diff
@@ -0,0 +1,69 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#pragma once
+#include <wil\result.h>
+
+// Define _PROC_THREAD_ATTRIBUTE_LIST as an empty struct because it's internal-only and variable-sized
+struct _PROC_THREAD_ATTRIBUTE_LIST {};
+
+struct MyProcThreadAttributeList
```

Comment:
NIT: Change this name to something more descriptive.  Or, at least get rid of "my"